### DISCUSSION TITLE
Show related threads to a person

### DISCUSSION THREAD

**Discussion by chenilim** [mattermost/community#55] [General Ideas]
I very often want to see:
* Recent threads person A participated in
* Recent threads person A and I both participated in
* Recent threads person A, B, C, etc. participated in

I *think* there's a way to do this with search, but can't figure it out. Ideally, I want to click on a user and then select "show recent threads", "show threads with A and me", etc.

Does anyone else run into this often?

**Comment by matthewbirtch** [comment]
Cool idea!

**Comment by esethna** [comment]
Nice. I think this extends to the idea of searching within the thread list. ie filter by participants or channel or time

I often finding myself scrolling the "all your threads" list to find a thread I'm looking for

cc// @matthewbirtch

**Comment by Willyfrog** [comment]
Maybe this could also appear in the user card that popups when you click on a user name?

Obviously not the threads, but a link to go get this info.

If this is a plugin it should be named [Troy McLure](https://en.wikipedia.org/wiki/Troy_McClure) 🤣

**Comment by chetanyakan** [comment]
A recommendation system like people who participated in this thread also participated.

**Reply by esethna** [reply]
Related to the idea of a Mattermost home screen: https://github.com/mattermost/community/discussions/62
